tests/js/server/aql/aql-queries-optimizer-limit-noncluster.js

```diff
@@ -28,44 +26,38 @@
 /// @author Copyright 2012, triAGENS GmbH, Cologne, Germany
 ////////////////////////////////////////////////////////////////////////////////
 
-var jsunity = require("jsunity");
-var internal = require("internal");
-var helper = require("@arangodb/aql-helper");
-var getQueryResults = helper.getQueryResults;
-var db = internal.db;
+const jsunity = require("jsunity");
+const internal = require("internal");
+const helper = require("@arangodb/aql-helper");
+const getQueryResults = helper.getQueryResults;
+const db = internal.db;
 
 ////////////////////////////////////////////////////////////////////////////////
 /// @brief test suite
 ////////////////////////////////////////////////////////////////////////////////
 
 function ahuacatlQueryOptimizerLimitTestSuite () {
-  var collection = null;
-  var docCount = 100;
-  var cn = "UnitTestsAhuacatlOptimizerLimit";
+  const cn = "UnitTestsAhuacatlOptimizerLimit";
```

Comment:
Ahuacatl

Reply:
It's just a name. It doesn't really matter IMO.